common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -246,6 +246,17 @@ stop_function() {
     [ "$DELETE_LOGS" = "true" ] && prplmesh_delete_logs
 }
 
+execute_beerocks_command() {
+    dbg "Executing beerocks cli command: $*"
+    if [ -e $PRPLMESH_BIN_DIR/beerocks_cli ]; then
+        output=$(${PRPLMESH_BIN_DIR}/beerocks_cli -c "$@")
+        return $?
```

Comment:
Why do you return the exit code of the operation instead of the actual output?

Reply:
The output is stored in the global `output` variable
while the exit code is returned
Isn't that proper code functionality?

Reply:
Well, you could return just an empty string in case of a failure..
You don't use the exit code anyway

Reply:
There is a validation of the bml exit code `if execute_beerocks_command "$bml_cmd" ; then`
If the beerocks_cli doesn't exist it would then return an error code of 1